### ISSUE TITLE
find method not passing parameters as expected

### ISSUE THREAD

**Issue by MordiSacks** [saintsystems/odata-client-php#50]
Hi,
when i call 
```php
$client->get('ENTITY(\'123456\')');
```
the url generated is 
`https://example/odata/ENTITY('123456')`
and I get the result

but when I try
```php
$client->from('ENTITY')->find('123456');
```
the url generated is 
`https://example/odata/ENTITY(123456)`
and I no results

**Comment by anderly** [comment]
For string-based identifiers, can you try: `$client->from('ENTITY')->find("'123456'");`?

Will look into this as we should support both numeric and string-based IDs.

**Comment by MordiSacks** [comment]
@anderly 
Thank you for your quick reply.

 when I try 
```php
$client->from('ENTITY')->find("'123456'");
```
the generated url is 
`https://example/odata/ENTITY(''123456'')` 
and I get a 404

This is the api I'm working with
https://prioritysoftware.github.io/restapi/request/

I believe it is 4.0
this is my first time working with OData
thank you.

**Comment by MordiSacks** [comment]
Ok, I think I found the issue,

`SaintSystems\OData\Query\Grammer::wrapKey()`  
checks for `is_numeric` and my string id is a number

**Comment by cmuench** [comment]
@MordiSacks Same problem if we try to connect to SAP sample service.

```
/sap/opu/odata/IWBEP/GWSAMPLE_BASIC/SalesOrderSet('0500000004')
```

The ID can have a leading zero.

**Comment by anderly** [comment]
@MordiSacks you're right. This should be `is_int`, not `is_numeric`. Looking at impact before making this change.

**Comment by anderly** [comment]
@cmuench, in your example, should there be quotes around '0500000004'?

**Comment by anderly** [comment]
Fixed in 9510dd0
